test/k8sT/Chaos.go

```diff
@@ -59,103 +53,211 @@ var _ = Describe("K8sChaosTest", func() {
+				helpers.DefaultNamespace,
+				fmt.Sprintf("-l zgroup=testDSClient"), 300)
+			Expect(err).Should(BeNil(), "Pods are not ready after timeout")
 
-		err = kubectl.CiliumEndpointWaitReady()
-		Expect(err).To(BeNil(), "Endpoints are not ready after timeout")
+			err = kubectl.CiliumEndpointWaitReady()
+			Expect(err).To(BeNil(), "Endpoints are not ready after timeout")
 
-		By("Checking connectivity before restarting Cilium")
-		PingService()
+			By("Checking connectivity before restarting Cilium")
+			PingService()
 
-		By("Deleting cilium pods")
-		res := kubectl.Exec(fmt.Sprintf("%s -n %s delete pods -l k8s-app=cilium",
-			helpers.KubectlCmd, helpers.KubeSystemNamespace))
-		res.ExpectSuccess()
+			By("Deleting cilium pods")
+			res := kubectl.Exec(fmt.Sprintf("%s -n %s delete pods -l k8s-app=cilium",
+				helpers.KubectlCmd, helpers.KubeSystemNamespace))
+			res.ExpectSuccess()
 
-		ExpectAllPodsTerminated(kubectl)
+			ExpectAllPodsTerminated(kubectl)
 
-		ExpectCiliumReady(kubectl)
-		err = kubectl.CiliumEndpointWaitReady()
-		Expect(err).To(BeNil(), "Endpoints are not ready after Cilium restarts")
+			ExpectCiliumReady(kubectl)
+			err = kubectl.CiliumEndpointWaitReady()
+			Expect(err).To(BeNil(), "Endpoints are not ready after Cilium restarts")
 
-		By("Checking connectivity after restarting Cilium")
-		PingService()
+			By("Checking connectivity after restarting Cilium")
+			PingService()
 
-		By("Uninstall cilium pods")
+			By("Uninstall cilium pods")
 
-		res = kubectl.DeleteResource(
-			"ds", fmt.Sprintf("-n %s cilium", helpers.KubeSystemNamespace))
-		res.ExpectSuccess("Cilium DS cannot be deleted")
+			res = kubectl.DeleteResource(
+				"ds", fmt.Sprintf("-n %s cilium", helpers.KubeSystemNamespace))
+			res.ExpectSuccess("Cilium DS cannot be deleted")
 
-		ExpectAllPodsTerminated(kubectl)
+			ExpectAllPodsTerminated(kubectl)
 
-		By("Checking connectivity after uninstalling Cilium")
-		PingService()
+			By("Checking connectivity after uninstalling Cilium")
+			PingService()
 
-		By("Install cilium pods")
+			By("Install cilium pods")
 
-		err = kubectl.CiliumInstall(helpers.CiliumDefaultDSPatch, helpers.CiliumConfigMapPatch)
-		Expect(err).To(BeNil(), "Cilium cannot be installed")
+			err = kubectl.CiliumInstall(helpers.CiliumDefaultDSPatch, helpers.CiliumConfigMapPatch)
+			Expect(err).To(BeNil(), "Cilium cannot be installed")
 
-		ExpectCiliumReady(kubectl)
+			ExpectCiliumReady(kubectl)
+
+			err = kubectl.CiliumEndpointWaitReady()
+			Expect(err).To(BeNil(), "Endpoints are not ready after timeout")
 
-		err = kubectl.CiliumEndpointWaitReady()
-		Expect(err).To(BeNil(), "Endpoints are not ready after timeout")
+			By("Checking connectivity after reinstalling Cilium")
+			PingService()
+		})
+	})
+
+	Context("Restart with long lived connections", func() {
+
+		var (
+			netperfManifest    = helpers.ManifestGet("netperf_deployment.yaml")
+			netperfPolicy      = helpers.ManifestGet("netperf-policy.yaml")
```

Comment:
Consistency amongst filenames is nice for tests - change netperf-policy.yaml --> netperf_policy.yaml